tao_compiler/mlir/disc/transforms/fusion_utils.cc

```diff
@@ -1162,6 +1162,16 @@ bool FusionStrategy::tryFuse(ShapeAnalysis& shapeAnalysis, FusionPattern& lhs,
   auto& operands = target.getOperands();
   auto& results = target.getResults();
 
+  // TODO(Yancey): support fusion with different reduction type
```

Comment:
it‘s better to push this check in BaseGpuFusionStrategy since the codegen logic for cpu is different？

Reply:
+1

Reply:
Agree. And the GPU stitch fusion also needs this checking.

Reply:
thanks, done.